src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1264,9 +1267,24 @@ void Ekf2::run()
 					_vel_d_innov_lpf = beta * _vel_d_innov_lpf + alpha * constrain(innovations.vel_pos_innov[2],
 							   -_vel_innov_spike_lim, _vel_innov_spike_lim);
 
+					// set the max allowed yaw innovaton depending on whether we are not aiding navigation using
+					// observations in the NE reference frame.
+					float yaw_test_limit;
```

Comment:
Can you move the definition of `filter_control_status_u _ekf_control_mask` in here, to minimize its scope, since it's only used here?

Reply:
OK